delatore/sources/influx.py

```diff
@@ -270,8 +270,7 @@ async def _get_status(self, metric):
         last_time_ms = None
         status = Status.FAIL
         if len(self.list_elapsed_time) == 0:
-            for seria in last_record.raw['series']:
-                self.list_hosts.append(seria['tags']['server'])
+            self.list_hosts = [seria['tags']['server'] for seria in last_record.raw['series']]
```

Comment:
This will replace existing `self.list_hosts`, not update it as it was before. Are you sure?